layers/+emacs/org/packages.el

```diff
@@ -544,12 +544,33 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
 (defun org/init-org-brain ()
+  (use-package ascii-art-to-unicode
+    :defer t
+    :init
+    (add-hook 'org-brain-after-visualize-hook #'aa2u-buffer))
```

Comment:
I have noticed that aa2u-buffer is not defined in `ascii-art-to-unicode` could you please double check?

Reply:
It looks like this was taken from the org-brain README, which was since updated to include a more comprehensive snippet:

https://github.com/Kungsgeten/org-brain#ascii-art-to-unicode

```elisp
(defface aa2u-face '((t . nil))
  "Face for aa2u box drawing characters")
(advice-add #'aa2u-1c :filter-return
            (lambda (str) (propertize str 'face 'aa2u-face)))
(defun aa2u-org-brain-buffer ()
  (let ((inhibit-read-only t))
    (make-local-variable 'face-remapping-alist)
    (add-to-list 'face-remapping-alist
                 '(aa2u-face . org-brain-wires))
    (ignore-errors (aa2u (point-min) (point-max)))))
(with-eval-after-load 'org-brain
  (add-hook 'org-brain-after-visualize-hook #'aa2u-org-brain-buffer))
```

Not sure if this belongs in Spacemacs - the bit with advising aa2u is somewhat of a hack and I would submit a PR upstream, but aa2u isn't on a public repo.

Reply:
Thanks for checking ... looks strange, I really cannot find where aa2u should be defined. Can it be that this is in `artist-mode` or in `picture-mode`? Anyway I agree with you the rendering related calls should be rather upstream when possible.

Reply:
It's defined in the ELPA `ascii-to-unicode` package if that's what you mean?